week_1/String/group_anagrams/group_anagrams.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param {string[]} strs An array of strings
+ * @return {string[][]} An array of string arrays, grouped according to whether they contain the same characters
+ */
+const groupAnagrams = (strs) => {
+	const groups = {};
+
+	strs.forEach((str) => {
+		// Sort the strings alphabetically
+		let key = str.split("").sort().join("");
```

Comment:
Whenever you go to call something like `strs`, `str`, `numbers`, `key`, `value`, etc, try to come up with something more descriptive. Every change helps a ton towards readability 
```suggestion
		const sortedString = word.split("").sort().join("");
```